addons/arsenal/stringtable.xml

```diff
@@ -1253,6 +1254,7 @@
             <Russian>Интегрирован тепловизор и осн.прицел.</Russian>
             <Korean>열화상과 주무기 내장</Korean>
             <French>Thermique et primaire intégrés</French>
+            <German>Thermal &amp; Primär integriert</German>
```

Comment:
"Primär integriert" versteht man denke ich nicht. Primary steht hier für die Hauptwaffe.

Reply:
Geändert in "Thermal und in Primärwaffe integriert"